README.md

```diff
@@ -28,6 +28,25 @@ $ dnf install python{2,3}-conu conu-doc
 
 Please visit [our documentation](http://conu.readthedocs.io/en/latest/installation.html) for more info on installation.
 
+## Docker container
+
+You can try conu also in the container, but you have to mount docker socket and use `--privileged` option:
+
+```
+docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock:z --privileged modularitycontainers/conu:0.2.0 python3
```

Comment:
Running in non-privileged mode: `docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock:z modularitycontainers/conu:0.2.0 make test-doc-examples` went OK in my system - aside from atomic mount example which always need superuser. I wonder what could make a difference. IMO we should investigate this more.

Reply:
Maybe @TomasTomecek should know that?

Reply:
```
$ man 2 mount
...
Appropriate privilege (Linux: the CAP_SYS_ADMIN capability) is required to mount filesystems.
...
```
+1 to Dominika, I would then just mention, that if they require to mount containers/images; they need to supply `--cap-add SYS_ADMIN`

Reply:
Ok, thanks.

Reply:
There is probably something bad on my site, but it does not work for me..;-(

Reply:
I've only changed  `--privileged` to `--cap-add SYS_ADMIN`. Is it ok?

Reply:
@TomasTomecek  I've made it work when the selinux is turned off...;-)

Reply:
I found this: https://danwalsh.livejournal.com/74095.html. This documentation PR needs a note explaining the way to deal with the issue you experienced.

Reply:
Ok, I've got it. Is the new version ok?